.gitmodules

```diff
@@ -354,3 +354,6 @@
 [submodule "lib/tlsf"]
 	path = lib/tlsf
 	url = https://github.com/espressif/tlsf.git
+[submodule "frozen/CircuitPython_AXP313A"]
+	path = frozen/CircuitPython_AXP313A
+	url = https://github.com/bill88t/CircuitPython_AXP313A
```

Comment:
oh, yeah, we should think about this.

We have previously allowed frozen modules that are in personal accounts and organization:
```
	url = https://github.com/pewpew-game/pew-pewpew-standalone-10.x.git
	url = https://github.com/python-ugame/circuitpython-stage.git
	url = https://github.com/pypewpew/pew-pewpew-lcd.git
	url = https://github.com/dahanzimin/circuitpython_lib.git
	url = https://github.com/elecfreaks/circuitpython_ef_music.git
	url = https://github.com/elecfreaks/circuitpython_picoed.git
	url = https://github.com/bablokb/circuitpython-pcf85063a
```

I do suspect that at a minimum you need to tag a numbered release in your repo, otherwise, our script to update frozen modules will fail (possibly without exiting with an error status, which would be our bug):
```
.../CircuitPython_AXP313A$ ~/src/circuitpython/tools/git-checkout-latest-tag.sh  
fatal: No names found, cannot describe anything.
```
I don't know if we've previously requested these authors to get the item into the Community Bundle, which might turn up other problems (you don't have the infrastructure that a cookiecutter-created library has, including stuff like pylint & black for code consistency; and I didn't see any readthedocs link but I didn't look too hard)

Please don't take this as a criticism, I appreciate the time that the work you are doing represents. But I want to make sure we aren't taking on something that will cause trouble down the line, so I mention these things.

Reply:
It's kinda my first time ever submitting a module at any capacity.
I up until this point, made modules for myself, and packaged them the ljinux way (CircuitMPY).

I honestly didn't even know cookiecutter exists up until now.
I've been kinda living under a rock when it comes to proper module distribution.

I will try to get to know the CP way, but currently I lack the experience of the dependencies.
I do not have any pylint or readthedocs experience I do not know how to set them up.